Patches/Drones/RimAtomics.xml

```diff
@@ -16,12 +16,12 @@
 					<value>
 						<ThingDef ParentName="PRF_MiniDroneStation">
 							<defName>PRF_MiniHelperAtomic</defName>
-							<label>Mini RimAtomic station</label>
+							<label>mini RimAtomic station</label>
 							<graphicData>
 								<texPath>Drones/MiniDroneStationAtomic</texPath>
 							</graphicData>
 
-							<description>A mini Atomic Workshop helper for RimAtomics to do maintain and fuel your reactors it can flick, haul, atomic, firefight</description>
+							<description>A mini atomic workshop helper for RimAtomics to maintain and fuel your reactors it can flick, haul, atomic, and firefight.</description>
```

Comment:
We already discussed these ", and" cases on Discord. I think it is better with the Oxford comma.

Reply:
Must have missed to mark that as resolved

Reply:
What about to change it into this? "A mini atomic workshop helper for RimAtomics to maintain and fuel your reactors, it can flick, haul, handle nuclear tasks, and firefight."